### ISSUE TITLE
Add coverage for India

### ISSUE THREAD

**Issue by PrathikGopal** [Hyundai-Kia-Connect/kia_uvo#634]
**Is your feature request related to a problem? Please describe.**
Finally was able to get a trace from the KIA mobile app for INDIA users , have the tracefile and will send it across

**Describe the solution you'd like**
Add coverage to India users, Have shared the traffic trace file to [fuatakgun@gmail.com](mailto:fuatakgun@gmail.com) .
Please let me know if you need more information.

**Comment by PrathikGopal** [comment]
@fuatakgun absolutely. Thank you for taking a look, currently running HA via RPI. But can run the same locally with your input. please let me know.

**Comment by PrathikGopal** [comment]
Just checking back @fuatakgun. Please let me know if I can add or contribute further towards this 😊🙏

**Comment by pk802** [comment]
@fuatakgun I am also interested in testing this if it works in India.

**Comment by fuatakgun** [comment]
@PrathikGopal , can you please share the logs again? sorry for missing this

**Comment by PrathikGopal** [comment]
> @PrathikGopal , can you please share the logs again? sorry for missing this

Sent it over to you 😊👍🏽

**Comment by satyambaba** [comment]
> well received

@fuatakgun, did you get a chance to look at this? Thanks for considering this. I also drive a Kia in India, please let me know if I can help in some way. Thanks a lot for your efforts.

**Comment by PrathikGopal** [comment]
@fuatakgun still awaiting eagerly 👍 incase you need the logs let me know.

**Comment by gowthamgowtham** [comment]
Library is updated to work with India region in _india_ branch in my repo. https://github.com/gowthamgowtham/hyundai_kia_connect_api.

I've tested for a Hyundai vehicle. Not sure if Kia needs any changes, may be the URLs? 

It was forked couple of months ago. Some cleanup might be necessary. The repo has a executable script `india.py` to test if it works for you.

**Comment by cdnninja** [comment]
Could you create a pull request?

**Comment by satyambaba** [comment]
> Library is updated to work with India region in _india_ branch in my repo. https://github.com/gowthamgowtham/hyundai_kia_connect_api.
> 
> I've tested for a Hyundai vehicle. Not sure if Kia needs any changes, may be the URLs?
> 
> It was forked couple of months ago. Some cleanup might be necessary. The repo has a executable script `india.py` to test if it works for you.

@gowthamgowtham - It would be great if you could please create a pull request for this. Really appreciate your help with this.

**Comment by gowthamgowtham** [comment]
Sorry I could not reply sooner. I will get to creating pull request sometime this weekend. Thank your for your patience!

Meanwhile, I am just curious to know if anyone in India had any success with this code?

**Comment by sajid2310** [comment]
@gowthamgowtham I can test this for Kia India however cant seem to find the india.py file in your repo?. Any idea

**Comment by PrathikGopal** [comment]
@sajid2310 heres the file, 

https://github.com/gowthamgowtham/hyundai_kia_connect_api/blob/india/india.py 

@gowthamgowtham  Will test it later today and update.

**Comment by PrathikGopal** [comment]
doesnt work for KIA, URL is different https://prd.in-ccapi.kia.connected-car.io:8080

**Comment by ohuc** [comment]
any updates?

**Comment by shaleenb** [comment]
Are there any updates on this? Will this be merged with master anytime soon?

**Comment by PrathikGopal** [comment]
@fuatakgun Thanks for your time on this, anyway i could be of help in speeding this up?

**Comment by ohuc** [comment]
+1

**Comment by cdnninja** [comment]
I encourage one of you to try take this on.   It was close to done here. https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/issues/499

**Comment by Nishant0127** [comment]
Hi! Has anyone got it working, ill be happy to assist, i have a HA and a Kia Carnival. Dont know python but can try and assist to get it working.

**Comment by nishant713** [comment]
+1
I've HA and Blue Link (Venue)

**Comment by cdnninja** [comment]
Still need someone to step up who is within the region and work on the above linked PR.

**Comment by cdnninja** [comment]
Thanks to @nishant713  this has been added to the master branch.  Next release will include it.
